cmd/modulectl/create/long.txt

```diff
@@ -14,21 +14,32 @@ The module config file is a YAML file used to configure the following attributes
 - name:             a string, required, the name of the module
 - version:          a string, required, the version of the module
 - channel:          a string, required, channel that should be used in the ModuleTemplate CR
-- manifest:         a string, required, reference to the manifest, must be a relative file name or URL
+- manifest:         a string, required, reference to the manifest, must be a URL
 
-- defaultCR:        a string, optional, reference to a YAML file containing the default CR for the module, must be a relative file name or URL
+- defaultCR:        a string, optional, reference to a YAML file containing the default CR for the module, must be a URL
 - mandatory:        a boolean, optional, default=false, indicates whether the module is mandatory to be installed on all clusters
 - resourceName:     a string, optional, default={NAME}-{CHANNEL}, the name for the ModuleTemplate CR that will be created
 - internal:         a boolean, optional, default=false, determines whether the ModuleTemplate CR should have the internal flag or not
 - beta:             a boolean, optional, default=false, determines whether the ModuleTemplate CR should have the beta flag or not
 - security:         a string, optional, name of the security scanners config file
 - labels:           a map with string keys and values, optional, additional labels for the generated ModuleTemplate CR
 - annotations:      a map with string keys and values, optional, additional annotations for the generated ModuleTemplate CR
+- manager: # an object, optional, module resource that indicates the installation readiness of the module
```

Comment:
What does the "#" mean?
For lines 27 to 36, would it be possible to align the indentation for the fields' descriptions so that they are in one line with the rest? :)

Reply:
so the # is the comment because if I just wrote the description without it, a formatting error is shown because I have the fields below